### ISSUE TITLE
Building problems

### ISSUE THREAD

**Issue by zgxsin** [ankitdhall/lidar_camera_calibration#101]
When using catkin_make to build the problem in ros-melodic, i have the following terminal output:
![image](https://user-images.githubusercontent.com/23395893/64846553-38f4d880-d60d-11e9-9de4-7fa3521fb857.png). Anyone having the same issue?

**Comment by SiddhantNadkarni** [comment]
Yes, I had similar issues. This is mostly a PCL - VTK build issue I believe. I got around this by compiling VTK 8.2 and PCL 1.9 from source.
